packages/ui/src/i18n/dictionaries/authenticator/en.ts

```diff
@@ -24,7 +24,7 @@ export const enDict: AuthenticatorDictionary = {
   'Enter your Birthdate': 'Enter your Birthdate',
   'Enter your code': 'Enter your code',
   'Enter your Confirmation Code': 'Enter your Confirmation Code',
-  'Enter your email': 'Enter your email',
+  'Enter your Email': 'Enter your Email',
```

Comment:
Have you run integration tests (including RN) on this change?

Reply:
Shouldn't they be running below? I think changes to ui package should probably trigger the RN tests?

Reply:
The job run on the initial commit got cancelled in favor of the merge commit which looks like doesn't have changes to relevant packages so RN e2es got skipped.